Source/Events.Processing/Filters/ValidateFilterByComparingStreams.cs

```diff
@@ -40,7 +39,10 @@ public class ValidateFilterByComparingStreams : IValidateFilterByComparingStream
         public async Task<FilterValidationResult> Validate<TFilterDefinition>(IFilterDefinition persistedDefinition, IFilterProcessor<TFilterDefinition> filter, CancellationToken cancellationToken)
             where TFilterDefinition : IFilterDefinition
         {
-            if (persistedDefinition == default) return new FilterValidationResult();
+            if (persistedDefinition == default)
```

Comment:
Nice that you made this class better as well. Since this is really similar to the new implementation could we do the same quality of life things here as well with reducing the LOC of this method by adding more methods?